ansible/inventories/bionic/group_vars/inventory-web/vars.yml

```diff
@@ -9,8 +9,7 @@ project_index_path: /usr/lib/ckan
 
 # db credentials
 ckan_db_server: "{{ postgresql_login_host }}"
-inventory_ckan_app_version: inventory-bionic
-inventory_ckan_ckanext_saml2_version: bionic-m2crypto
+inventory_app_repo_branch: bionic
```

Comment:
This can be removed, the variable in ansible/inventories/bionic/group_vars/all/vars.yml will get picked up across all Ansible groups in the bionic inventory. Having it here is redundant.

Reply:
done